_data/navigation.yml

```diff
@@ -476,6 +476,8 @@ guides:
           url: /guides/user/managed-delivery/resource-status/
         - title: API
           url: /guides/user/managed-delivery/api/
+        - title: Integrate Your CI
```

Comment:
Suggestion:
```suggestion
        - title: Integrate Your CI Builds
```

Reply:
I'm not sure that we should limit it to `builds` only. If a developer will integrate his CI provider, he can potentially do much more :)